infrastructure/src/main/java/org/corfudb/infrastructure/DynamicTriggerPolicy.java

```diff
@@ -43,6 +46,19 @@ private boolean shouldForceTrigger(CorfuStore corfuStore) {
         return false;
     }
 
+    @VisibleForTesting
+    public DistributedCheckpointerHelper getDistributedCheckpointerHelper(CorfuStore corfuStore) throws Exception {
```

Comment:
Can we use CorfuService::getDistributedCheckpointerHelper instead of writing a duplicate one?

Reply:
done